packages/drivers/odsp-driver/src/odspUtils.ts

```diff
@@ -295,6 +296,13 @@ export function toInstrumentedOdspTokenFetcher(
                     throwOdspNetworkError("tokenIsNull", fetchTokenErrorCode, undefined, undefined, { method: name });
                 }
                 return token;
+            }, (error) => {
+                const tokenError = wrapErrorAndLog(
+                    error,
+                    (errorMessage) => createOdspNetworkError("tokenFetcherFailed", errorMessage, fetchTokenErrorCode),
```

Comment:
Note that this is change in contract, and it has back compat considerations, as existing (in prod) bits will get latest version of driver through dynamic loading, and may not expect this new error code.
It might be fine (as handling of unknown code might be the same as handling of errors with no code), but likely worth calling out in breaking.md, allowing consumers to make decisions on how to consume these changes (and possibly deploy change that knows how to handle these errors before they consumer new version of FF)